lib/Github/Api/Deployment.php

```diff
@@ -66,6 +66,25 @@ public function create($username, $repository, array $params)
         return $this->post('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/deployments', $params);
     }
 
+    /**
+     * Delete a deployment for the given username and repo.
+     *
+     * @link https://docs.github.com/en/rest/reference/repos#delete-a-deployment
+     *
+     * Important: Deployments can only be deleted when in inactive state
+     * @see updateStatus
+     *
+     * @param string $username   the user who owns the repo
+     * @param string $repository the name of the repo
+     * @param int    $id         the id of the deployment
+     *
+     * @return mixed null on success, array on error with 'message'
+     */
+    public function remove($username, $repository, $id)
```

Comment:
One last thing, can you add typehints to the parameters and remove the superfluous `@param` phpdocs

Reply:
changed per feedback.

I am wondering as other methods inside the same file don't use native typehints and have these phpdocs.. so I was trying to be consistent with the rest of the file.

Reply:
Yes, that was good. But We're planning on adding typehints to all methods, but for existing methods this might be a BC break. So we can already add them to new methods, so it's less work afterwards 😄